package.json

```diff
@@ -12,7 +12,8 @@
     "node-sass": "^4.14.1",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
-    "react-scripts": "^3.4.3",
+    "react-magnifier": "^3.0.4",
+    "react-scripts": "^3.4.3"
```

Comment:
this should be a `"^3.4.3",` <--- with the comma, do you mind changing that quickly?